ReactNative/ReactTwitter/core/views/debug/push-notifications-screen.js

```diff
@@ -35,8 +35,9 @@ var PushNotificationsScreenView = React.createClass({
         <Button
           style={styles.button}
           styleDisabled={styles.button_disabled}
-          onPress={this._buttonClicked}> Send push notification</Button>
+          onPress={this._buttonClicked}> Send Android push notification</Button>
```

Comment:
This button send a push notification to test the functionality. 
This feature has been de-scoped and it will be probably deleted from the debug screen has we don't want to deploy private keys nor certificates.